ci/jenkins/jobs/projects.yaml

```diff
@@ -377,6 +377,195 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+      - '{test_name}-for-pull-request':
+          test_name: nephe-test-e2e-eks-with-agent
+          node: 'nephe'
+          description: 'This is the {test_name}.'
+          branches:
+            - ${{sha1}}
+          builders:
+            - deploy-vm-with-agent-and-run-eks-tests-for-nephe
+          trigger_phrase: .*/nephe-test-e2e-(eks-with-agent|with-agent|all).*
```

Comment:
The trigger phrase is not unique. 
`nephe-test-e2e-eks-with-agent` is triggering jobs for nephe-test-e2e-eks, nephe-test-e2e-eks-with-agent and nephe-test-e2e-aks-with-agent.

Reply:
Changed to nephe-test-e2e-with-agent-aks(eks).